.github/workflows/nightly.yml

```diff
@@ -24,80 +19,267 @@ jobs:
           restore-keys: ${{ runner.os }}-m2
 
       - name: Build Archetype
-        run: mvn clean install --file archetype/pom.xml
+        run: mvn clean install --file archetype/pom.xml  
 
-      - name: Run Archetype for EE 8, SE 8
-        run: |
-          mvn archetype:generate -DinteractiveMode=false -DaskForDefaultPropertyValues=false -DarchetypeGroupId=org.eclipse.starter -DarchetypeArtifactId=jakarta-starter -DarchetypeVersion=2.3.0-SNAPSHOT -DjakartaVersion=8 -Dprofile=full -DjavaVersion=8 -DoutputDirectory=app -Dgoals="clean package"
-          rm -rf app
-          
-      - name: Run Archetype for EE 8 Web Profile, SE 8
+      - name: Set up Java SE 8
+        uses: actions/setup-java@v3
+        with:
+          distribution: 'temurin'
+          java-version: 8
+
+      - name: Run Archetype for EE 9.1 Full Profile, SE 8, Open-liberty
```

Comment:
Please fully describe the case. Is it using Docker or not? Also, please use "Open Liberty" instead of "Open-liberty".